source-control/.gitconfig

```diff
@@ -8,14 +7,20 @@
 
   aa = add . # Stage all changes
   amend = commit --amend -C HEAD # Amend the previous commit message
+  b = branch
   ba = branch --all # Show all local and remote branches
+  bd = branch -D
```

Comment:
This is the same as `d = branch -D # Hard delete branch (whether it has unpushed changes or not)`